TraineeTasks/RegexTasks.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace TraineeTasks
+{
+    public class RegexTasks
+    {
+        /// <summary>
+        /// Find words in a string that contain the letter h, followed by 0 or 1 letter u.
+        /// </summary>
+        /// <param name="str"></param>
+        /// <returns>Returns a string of found words.</returns>
+        public string FindWords(string str)
+        {
+            if (String.IsNullOrEmpty(str))
+            {
+                throw new ArgumentException("The string cannot be null or empty string");
+            }
+
+            var regex = new Regex(@"\w*h(u{0}|u{1})\w*", RegexOptions.IgnoreCase);
+            var matches = regex.Matches(str);
+            List<string> words = new List<string>();
+
+            foreach (Match item in matches)
+            {
+                if (!item.Value.Contains("uu"))
+                {
+                    words.Add(item.Value);
+                }
+            }
+
+            if (words.Count > 0)
+            {
+                return words.Aggregate(((i, j) => i + ", " + j));
+            }
```

Comment:
This part incorrect. There is no reason to return just a string, it must be collection